indico/modules/events/models/events.py

```diff
@@ -322,6 +328,11 @@ def participation_regform(self):
         return self.registration_forms.filter_by(is_participation=True, is_deleted=False).first()
 
     @property
+    def published_registrations(self):
```

Comment:
`@memoize_request` - you already call it multiple times in the same template and right now that sends multiple queries